InitializeRepo/initialize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
```

Comment:
Add a `set -euo pipefail` here to fail fast if there any errors, especially to avoid making any weird irreversible changes later.

Reply:
Added